Rewrite AAR configuration howto guide and add AAR command reference (#437)
canonical/anbox-cloud-docs: howto/aar/configure.md

    juju switch <model containing ams>
    juju relate ams <offer name>

## Configuring AAR for the Appliance

If you are using the appliance, you must register the clients manually. Adding clients manually requires access to the machines hosting AMS and the AAR.

Install the snap:

>     sudo snap install --channel= <channel> aar

Reviewer: Nitpick:
```suggestion
    sudo snap install --channel=<channel> aar
```

> Add the suitable snap version instead of `<channel>`. For example, it could be `1.27/stable`. For a list of all snap versions available, you can run `snap info aar`.

Reviewer: This reads a bit odd. Shouldn't we tell users what to do directly? e.g.:

```suggestion
Replace `<channel>` with a suitable snap version. For example, it could be `1.27/stable`. For a list of all snap versions available, you can run `snap info aar`.
```

Wdyt?
